test/integration/connect/envoy/case-grpc/verify.bats

```diff
@@ -19,7 +19,7 @@ load helpers
 }
 
 @test "s1 proxy should be sending gRPC metrics to statsd" {
-  run retry_default must_match_in_statsd_logs 'envoy.cluster.grpc.PingServer.total'
+  run retry_default must_match_in_statsd_logs 'envoy.cluster.default.dc1.internal.*.consul.grpc.PingServer.total'
```

Comment:
This is being passed to grep IIRC so it works but is a bit confusing that `.*` is not really a glob but only working because we happen to have `.*`.